packages/massa-web3/src/experimental/basicElements/keys.ts

```diff
@@ -199,6 +200,7 @@ export class PrivateKey {
 export class PublicKey {
   public bytes: Uint8Array
 
+  // eslint-disable-next-line max-params
```

Comment:
I feel like we'll need to add this statement on many functions, especially constructors.
Maybe we can set a higher number for max-params but otherwise I would consider removing this rule due to overhead.

Reply:
You're right, but it's on purpose.
Here, the constructor is protected, which means that an end-user won't call it directly. This is why I have accepted disabling this rule. If the method were public, I would have reorganized the code.
The underlying idea is to have a cleaner SDK by reducing the number of arguments in the method signatures.